projects/js-packages/connection/components/manage-connection-dialog/index.jsx

```diff
@@ -82,22 +139,43 @@ const ManageConnectionDialog = props => {
 									'jetpack-connection-js'
 								) }
 							</Text>
-							<ManageConnectionActionCard
-								title={ __( 'Transfer ownership to another admin', 'jetpack-connection-js' ) }
-								link={ getRedirectUrl( 'calypso-settings-manage-connection', {
-									site: window?.myJetpackInitialState?.siteSuffix,
-								} ) }
-								key="transfer"
-								action="transfer"
-							/>
-							<ManageConnectionActionCard
-								title={ __( 'Disconnect Jetpack', 'jetpack-connection-js' ) }
-								onClick={ openDisconnectDialog }
-								key="disconnect"
-								action="disconnect"
-							/>
+							{ isCurrentUserAdmin &&
+								connectedUser.currentUser?.isConnected &&
+								connectedUser.currentUser?.isMaster && (
+									<ManageConnectionActionCard
```

Comment:
I know this is the same functionality as before, but I was wondering if we should open a new window when the user clicks this link? The "External link" icon generally means a new window will open as well in my experience

![image](https://github.com/user-attachments/assets/4e2c7da5-7e44-4454-98bc-1139af6c603e)

Reply:
Good thought, yes I think that makes sense